specs/lib/loadObjSpec.js

```diff
@@ -535,6 +536,18 @@ describe("loadObj", () => {
     expect(baseColorTexture.source).toBeDefined();
   });
 
+  it("loads an obj which origins from Tinkercad", async () => {
+    /**
+     * Tinkercad formats the obj files with both spaces and tabs. However, several obj readers know
+     * how to deal with those obj files, so we want this library to also do that
+     */
+    const data = await loadObj(objFromTinkerCad, options);
+    const primitive = getPrimitives(data)[0];
+    expect(primitive.positions.length / 3).toBe(393);
+    expect(primitive.normals.length / 3).toBe(0);
+    expect(primitive.uvs.length / 2).toBe(0);
+  });
```

Comment:
The test file could be a little smaller and more generic. Could you copy `box` and make a `box-with-tabs`, replacing the spaces with tabs? Otherwise the test looks good.